riocli/apply/explain.py

```diff
@@ -14,9 +14,28 @@
 from pathlib import Path
 
 import click
+import yaml
 from click_help_colors import HelpColorsCommand
 
 from riocli.constants import Colors, Symbols
+from riocli.utils import tabulate_data
+
+
+@click.command(
+    'list-examples',
+    cls=HelpColorsCommand,
+    help_headers_color=Colors.YELLOW,
+    help_options_color=Colors.GREEN,
+    help='List all examples supported in rio explain command'
+)
+def list_examples() -> None:
+    path = Path(__file__).parent.joinpath('manifests')
```

Comment:
todo: Please make sure this runs when installing from wheel. Usually it will work locally since the files are present there, so I would suggest create a docker container with Python, copy the wheel there, install and try it.

Reply:
This will run when installing from wheel since `rio explain` also refers to the same location.
https://github.com/rapyuta-robotics/rapyuta-io-cli/blob/8ea304b914e5708d2ce8cb87ac9c88de11ff7b7c/riocli/apply/explain.py#L36C10-L36C59